Translating readme and specifying (#22)
DenisGuiraudet/ICE_supercopter: README.md

# Supercopter

M1 ICE project
School year 2018 - 2019

Chopper renting simulator
## Team
Denis Guiraudet, Matthieu Orriere, Arnaud Sibade and last but not least Alexandre Pausé

## Project description

### Slack 
Join us on [Slack](https://courscollab2018.slack.com/)
Channel : team_supercopter

### Specs 

> The project specs are specify just here in this Readme

Reviewer: ```suggestion
The project specs are specified just here in this Readme
```

### Following
 	We chose to manage the tasks and issues with GitHub Project

> 	PR management make with a link to issues

Reviewer: ```suggestion
	PR management made with a link to issues
```

### Repository
https://github.com/DenisGuiraudet/supercopter.git

### IC
Travis : https://travis-ci.com/DenisGuiraudet/supercopter

> Knowledge of this techno by some group members

Reviewer: ```suggestion
Why ? Previous knowledge of this techno by some group members
```

### Build

> This is going to evolve and change

Reviewer: ```suggestion
```

Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.

### Deployment

> TO DO

Reviewer: ```suggestion
```

We don't have any hosting for now

### Environement Dev
Docker

### Techno
 - Angular.io / SASS / html5

> 	Knowledge of Angular by some group members and want to improve our skill

Reviewer: ```suggestion
Why ?	Knowledge of Angular by some group members and want to improve our skill
```

 - mongoDB
	MongoDB is a simple database adapte to link with Angular

### Code Style
	TS lint

> 	We choose to use TS lint cause is the most used linter for TypeScript

Reviewer: ```suggestion
Why ?	We chose to use TS lint cause is the most used linter for TypeScript
```
